common/js/src/logging/logging.js

```diff
@@ -48,46 +48,39 @@ goog.require('goog.log.Logger');
 
 goog.scope(function() {
 
-/** @const */
-var LOGGER_SCOPE = 'GoogleSmartCard';
+const LOGGER_SCOPE = 'GoogleSmartCard';
 
 /**
  * The logging level that will be applied to the root logger (and therefore
  * would be effective for all loggers unless the ones that have an explicitly
  * set level).
- * @type {!goog.log.Level}
```

Comment:
What is the reason to remove the type here?

Reply:
I assume that the compiler will be able to deduce the type automatically, based on the fact that it's a constant and the expression isn't too complex (e.g., without calls to functions that can return values of different types).